src/Leaflet.Editable.js

```diff
@@ -1882,6 +1882,7 @@
         this.on('add', this._onEditableAdd);
     };
 
+    var flat = L.LineUtil.isFlat || L.LineUtil._flat || L.Polyline._flat;  // <=> 1.1 compat.
```

Comment:
I think that since this is a `boolean` it should be named as such. `isFlat` seems to be a better name and the same name that is used in `L.LineUtil` starting from the `Leaflet` `1.2.0`